extensions/amp-youtube/1.0/amp-youtube.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {VideoBaseElement} from '../../amp-video/1.0/base-element';
+import {Youtube} from './youtube';
+
+/** @const {string} */
+const TAG = 'amp-youtube';
+
+class AmpYoutube extends VideoBaseElement {}
+
+/** @override */
+AmpYoutube['Component'] = Youtube;
+
+/** @override */
+AmpYoutube['props'] = {
+  'autoplay': {attr: 'autoplay'},
+  'loop': {attr: 'loop'},
+  'videoid': {attr: 'data-videoid'},
+  'liveChannelid': {attr: 'data-live-channelid'},
+  'dock': {attr: 'dock'},
+  'credentials': {attr: 'credentials'},
+  'params': {attrPrefix: 'data-param-'},
+};
+
+/** @override */
+AmpYoutube['passthrough'] = true;
```

Comment:
This will yell at you since it doesn't make sense with `VideoBaseElement`. 

![image](https://user-images.githubusercontent.com/254946/98863390-1df24400-241d-11eb-9ee1-cd97ee6a33bd.png)

You should just remove it.

```suggestion
```